### DISCUSSION TITLE
ui netbeans templete not change to Universal-G-Code-Sender

### DISCUSSION THREAD

**Discussion by annopnod** [winder/Universal-G-Code-Sender#2580] [General]
![image](https://github.com/user-attachments/assets/abbd22c0-e8ca-4768-9dbe-f1a00b63f428)

**Comment by annopnod** [comment]
netbeans ide?????

**Reply by breiler** [reply]
UGS is built on top of the NetBeans IDE project. We have a config file in the project that should have overridden this text but that stopped working a while back when we upgraded the NetBeans version. It had a low priority so I didn't want to spend too much time on fixing this.
